### PR TITLE
return reputation rewarded from Redeem function

### PR DESCRIPTION
**Pull request by dkent600**
The dutchx app wants to display to the user how much reputation the user will be rewarded.  This PR change greatly facilitates obtaining this value after the locking period has ended and before the rep has been redeemed, and it keeps within the contract the logic for computing reputation.

(The amount of reputation rewarded after redeeming will have to come from the Redeem events.)

### CODE REVIEW

File: contracts/schemes/Auction4Reputation.sol [hunk 1]
```diff
@@ -86,9 +86,9 @@ contract Auction4Reputation is Ownable {
      * @dev redeem reputation function
      * @param _beneficiary the beneficiary to redeem.
      * @param _auctionId the auction id to redeem from.
-     * @return bool
+     * @return uint reputation rewarded
      */
-    function redeem(address _beneficiary, uint _auctionId) public returns(bool) {
+    function redeem(address _beneficiary, uint _auctionId) public returns(uint) {
```
Comment (orenyodfat) [inline contracts/schemes/Auction4Reputation.sol]: function redeem(address _beneficiary, uint _auctionId) public returns(uint reputation)

File: contracts/schemes/Locking4Reputation.sol [hunk 2]
```diff
@@ -43,9 +43,9 @@ contract Locking4Reputation {
     /**
      * @dev redeem reputation function
      * @param _beneficiary the beneficiary for the release
-     * @return bool
+     * @return uint reputation rewarded
      */
-    function redeem(address _beneficiary) public returns(bool) {
+    function redeem(address _beneficiary) public returns(uint) {
```
Comment (orenyodfat) [inline contracts/schemes/Locking4Reputation.sol]: function redeem(address _beneficiary) public returns(uint reputation)

and remove reputation definition inside the function.

Comment (dkent600) [reply contracts/schemes/Locking4Reputation.sol]: done: https://github.com/daostack/arc/pull/566/commits/2418b4276e2160c1936b493be377d5631215430c

File: contracts/schemes/Auction4Reputation.sol (hunk 1 above)
Comment (dkent600) [reply contracts/schemes/Auction4Reputation.sol]: done: https://github.com/daostack/arc/pull/566/commits/2418b4276e2160c1936b493be377d5631215430c

**Pull request merged**